### ISSUE TITLE
Injectus does not perform open redirect when no params.

### ISSUE THREAD

**Issue by JeffreyShran** [BountyStrike/Injectus#3]
![image](https://user-images.githubusercontent.com/10995361/104309348-ec9ffe00-54c9-11eb-8137-d3f6cc41392d.png)

Debian 10
Python 3.7

Injectus appears to not do anything when there are no parameters provided in open redirect mode which is contradictory to the readme. I'm happy to help test or clarify.

**Comment by dubs3c** [comment]
Hmm, strange. I haven't worked on this project for some time, but I have been meaning to update it :D If you are not in a hurry I can take a look as well during the week.

**Comment by JeffreyShran** [comment]
No, please don't rush on my behalf. I'm appreciative of you putting it out there and I was simply giving it a test drive. 

Please let me know if you need me to test or anything. I'm not a coder but I'm a quick learner.
